dialogflow/stopwatch/README.md

```diff
@@ -4,7 +4,7 @@ This app demonstrates how to make streaming gRPC connections to the [Dialogflow
 
 ## Prerequisites
 - Credentials for calling the Dialogflow API
-- An OSX machine or emulator
+- An OSX machine or simulator
 - [Xcode 9.1][xcode] or later
 - [Cocoapods][cocoapods] version 1.0 or later
 
```

Comment:
Rename `Prepare your Google Cloud account` --> `Setup`

Reply:
What steps do they need to perform in the `SETUP-SERVICE` directory? 
Those should be outlined as well if they need to run any of those.

Reply:
Remove `## Enable Dialogflow` (redundant)

Reply:
Under `## Running the app` They should first finish setting up the app. 

Looks like most of these steps except the last one should be removed.

Reply:
Yes, "./INSTALL-COCOAPODS" It will install everything including beta libraries as well for Dialogflow.